cmd/acikubectl/cmd/debug.go

```diff
@@ -165,72 +168,93 @@ func clusterReport(cmd *cobra.Command, args []string) {
 
 	cmds := []reportCmdElem{
 		{
-			name: "cluster-report/logs/controller/acc.log",
-			args: accLogCmdArgs(systemNamespace),
+			name:           "cluster-report/logs/controller/acc.log",
+			args:           accLogCmdArgs(systemNamespace),
+			skipOutputFile: false,
```

Comment:
Isn't the default value for a bool always false? If so, it might not be required to initialize with false in most cases below?